src/content/docs/release-notes/docs-release-notes/docs-6-7-2024.mdx

```diff
@@ -0,0 +1,52 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-07'
+version: 'ersion: May 31-June 6, 2024'
+---
+
+### New docs
+
+* A new [OpenTelemetry data in New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-data-overview) doc is an index to help you find the OpenTelemetry info you're looking for.
+
+### Major changes
+
+* We've combined all our [.NET API](/docs/apm/agents/net-agent/net-agent-api/net-agent-api) docs so now you can see them all in one place.
```

Comment:
@paperclypse the link to the .NET API doc showed the same list of API calls as the one that's in public. Could you clarify what the differences here are?

Reply:
I expect because this change was made several days ago